uri.opam

```diff
@@ -15,21 +15,16 @@ tags: [
   "org:mirage"
   "org:xapi-project"
 ]
-build: [ "ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%" "--tests" "false" ]
-build-test: [
-  [ "ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%" "--tests" "true" ]
-  [ "ocaml" "pkg/pkg.ml" "test" ]
-]
+build: [ "jbuilder" "build" "@install" ]
+build-test: [ "jbuilder" "build" "@runtest" ]
 depends: [
   "base-bytes"
-  "ocamlbuild" {build}
   "ocamlfind" {build}
+  "jbuilder" {build}
   "ounit" {test & >= "1.0.2"}
-  "ppx_deriving"
```

Comment:
When using jbuilder, this is not necessary. I don't know the exact details though, but jbuilder takes care of this without the need for ppx_deriving.